docs/faq.md

```diff
@@ -38,11 +38,9 @@ Go to <i class="uil uil-cog"></i> *Configuration* → <i class="uil uil-setting"
 !!! seealso alert alert-info "See also"
     [Configure data retention](how-to/configure.md#data-retention)
 
-## How often are NGINX logs rotated?
+## How are PMM Server logs rotated?
 
-Daily.
-
-PMM Server runs `logrotate` daily to rotate NGINX logs, keeping up to ten of the most recent log files.
+PMM Server embeds multiple components, like Victoria Metrics, Query Analytics, grafana, managed, postgresql, clickhouse, etc (components). All PMM Server component logs are managed and rotated with `supervisord`. Each particular component has its own log rotation config within the `supervisord.conf` file, which defines both the maximum size of a log file and the number of log files to keep. The log rotation takes place once the log file reaches its maximum size. 
```

Comment:
> config within the `supervisord.conf`

We also have a lot of configs in /etc/supervisor.d/ I think it makes sense to add notes about it.

Reply:
Thanks, updated!